src/app/shared/components/validate-json-or-yaml/component.spec.ts

```diff
@@ -0,0 +1,230 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ComponentFixture, TestBed, waitForAsync} from '@angular/core/testing';
+import {MatFormFieldModule} from '@angular/material/form-field';
+
+import {ValidateJsonOrYamlComponent} from './component';
+
+describe('ValidateJsonOrYamlComponent', () => {
+  let component: ValidateJsonOrYamlComponent;
+  let fixture: ComponentFixture<ValidateJsonOrYamlComponent>;
+
+  beforeEach(waitForAsync(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ValidateJsonOrYamlComponent],
+      imports: [MatFormFieldModule],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ValidateJsonOrYamlComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('Template', () => {
```

Comment:
which component template? see this comment: https://github.com/kubermatic/dashboard/pull/4614/files#r905887082

Reply:
https://github.com/kubermatic/dashboard/pull/4614#discussion_r905913940

Reply:
nested `describe` block removed